wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/js/iframe_nav.js

```diff
@@ -5,7 +5,7 @@ document.addEventListener('DOMContentLoaded', () => {
     .toLocaleLowerCase()
     .replace(openverseSubpath, '')
     .replace(/^\/$/, ''); // Remove Openverse site subpath
-  iframePath = `${openverseUrl}${iframePath}${query}`; // Add domain and query
+  iframePath = `${openverseUrl}${localeSlug ? `/${localeSlug}` : ''}${iframePath}${query}`; // Add domain and query
```

Comment:
```suggestion
  iframePath = `${openverseUrl}${localeSlug ? `${localeSlug}` : ''}${iframePath}${query}`; // Add domain and query
```

Reply:
This slash causes the `openverse_embed` url have 2 slashes: `<iframe id="openverse_embed" src="http://localhost:8443//ru">`.
Without it, everything works, and if I use the branch from the[ translation banner fix PR](https://github.com/WordPress/openverse-frontend/pull/904), I can see the banner!

Reply:
Huh, I don't see that locally. For example, if I change the `get_locale` call to just be `pt_BR` I see this logged:

```
Navigating iframe to https://search-staging.openverse.engineering/pt-br
```

Reply:
Ah, I think this depends on what you set the `ov_embed_url` value to in customizer. If you add a trailing slash to that then it doubles the slash.

The default value does not include a trailing slash and I've just been testing by changing the subdomain to `search-staging` so that's what I haven't seen the double slash locally.